dotnet/src/IntegrationTests/TestHelpers.cs

```diff
@@ -27,16 +26,16 @@ internal static void ImportSampleSkills(IKernel target)
             "QASkill");
     }
 
-    internal static IDictionary<string, ISKFunction> GetSkills(IKernel target, params string[] skillNames)
+    internal static IDictionary<string, ISKFunction> GetPlugins(IKernel target, params string[] pluginNames)
```

Comment:
Ah, can you change to GetOrImportPlugins or something like that, to indicate that it's actually doing the import work too?

Reply:
Renamed to `ImportSamplePlugins`